lib/blog-feed-loader.js

```diff
@@ -1,2 +1,6 @@
-// TODO: Replace this with a real implementation.
-module.exports = require('../test/browser/stub-blog-feed-loader');
+if (process.env.NODE_ENV !== 'production' &&
+    require('./config').IN_TEST_SUITE) {
```

Comment:
I'm not a huge fan of doing this test suite conditional code here, but it makes it a lot easier to actually test our code. Bleh. We can probably revisit it and figure out a cleaner solution later, if nothing else.